webpack.config.js

```diff
@@ -4,7 +4,11 @@ const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
 const FixStyleOnlyEntriesPlugin = require('webpack-fix-style-only-entries');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const MonacoWebpackPlugin = require('monaco-editor-webpack-plugin');
-const PostCSSPresetEnv = require('postcss-preset-env');
+const PostCSSPresetEnv = () => require('postcss-preset-env')({
+  features: {
+    'system-ui-font-family': false,
+  }
+});
```

Comment:
I think we can put this config into the `postcss-loader` options. That way, it'll be easier to convert this file to ESM syntax later. Keep note that there are two sections, one for CSS and one for Less.

```js
{
  loader: 'postcss-loader',
  options: {
    plugins: () => [
      PostCSSPresetEnv({
        features: {
          'system-ui-font-family': false,
        },
    }),
    ],
    sourceMap: true,
  },
},
```

Reply:
I was about to do this:
```js
const PostCSSPresetEnv = (args) => {
  args = {...args};
  return require('postcss-preset-env')({...args,
    features: {...args.features,
      'system-ui-font-family': false,
    }
  });
};
```
I'm not sure about the link between ESM syntax and having this at two places.

Reply:
With ESM one would convert

```js
const PostCSSPresetEnv = require('postcss-preset-env');
```

to 

```js
import PostCSSPresetEnv from 'postcss-preset-env';
```

meaning the import statement has to stand alone, so I think it's better to leave it a simple `require` statement. You can optionally define the config on the top-level scope in a variable.

Reply:
https://github.com/CL-Jeremy/gitea/commit/a74795d74d14d59f29fd371ee9901872d8c79145

Reply:
Nice, didn't know that trick. The only thing that worries me a bit is the `esm` seems to lean on the unmaintained side, but I guess we can eventually remove it once webpack supports node's ESM implementation.

Reply:
Change looks good besides this. Can you either replicate the `postCSSPlugins` variable usage in this PR or alternatively also merge the ESM conversion into here?